tools/packaging/packaging.psm1

```diff
@@ -106,6 +110,14 @@ function Start-PSPackage {
         }
     }
 
+    if($Configuration -ine 'Release' -and $NameSuffix)
+    {
+        $NameSuffix = "$Configuration-$NameSuffix"
+    }
+    elseif ($Configuration -ine 'Release') {
+        $NameSuffix = $Configuration
```

Comment:
Just want to double check -- we don't change the existing names of the release packages, right?